docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -279,212 +216,237 @@ server, DHCP server, RADIUS server) using:
-
-[options="compact"]
-* Inline
-* Out-of-band using SNMP or RADIUS
-* Hostpot-style (or Web Auth)
-* RADIUS only
-
-It is also possible to combine enforcement modes. For example, you could use the out-of-band mode on your wired switches, while using the inline mode on your old WiFi access points.
-
-The following sections will explain these enforcement modes. If you decide to use the inline mode, please refer to the PacketFence Inline Deployment Quick Guide using ZEN for a complete configuration example. If you decide to use the out-of-band mode, please refer to the PacketFence Out-of-Band Deployment Quick Guide using ZEN
-
-Technical introduction to Inline enforcement
---------------------------------------------
-
-Introduction
-~~~~~~~~~~~~
+Getting Started
+---------------
 
-Before the version 3.0 of PacketFence, it was not possible to support unmanageable devices such as entry-level consumer switches or access-points. Now, with the new inline mode, PacketFence can be use in-band for those devices. So in other words, PacketFence would become the gateway of that inline network, and NAT or route the traffic using IPTables/IPSet to the Internet (or to another section of the network). Let see how it works.
+Now that PacketFence is installed, it needs to be configured. The PacketFence web-based configuration interface will automatically be started. 
 
-Device Configuration
-~~~~~~~~~~~~~~~~~~~~
-No special configuration is needed on the unmanageable device. That's the beauty of it. You only need to ensure that the device is "talking" on the inline VLAN. At this point, all the traffic will be passing through PacketFence since it is the gateway for this VLAN.
+This section will guide through configuring PacketFence as a simple RADIUS server. PacketFence will provide 802.1X support through Microsoft Active Directory and a Cisco 2960 access switch will be configured to integrate with PacketFence. The 802.1X client will be a Microsoft Windows 7 computer, connected of course on the wired network in the Cisco 2960 access switch. The following architecture diagram shows the interconnection of all components for our example:
 
-Access control
-~~~~~~~~~~~~~~
+image::docs/images/getting-started.png[scaledwidth="65%",alt="Getting Started"]
 
-The access control relies entirely on IPTables/IPSet. When a user is not registered, and connects in the inline VLAN, PacketFence will give him an IP address. At this point, the user will be marked as unregistered in the ipset session, and all the Web traffic will be redirected to the captive portal and other traffic blocked. The user will have to register through the captive portal as in VLAN enforcement. When he registers, PacketFence changes the device´s ipset session to allow the user's mac address to go through it.
+Going through the configurator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Limitations
-~~~~~~~~~~~
+First open PacketFence's configurator - you can access it from https://@ip_of_packetfence:1443/configurator
```

Comment:
I'd probably give the command to figure out the IP address using the CLI just as a safety net